### ISSUE TITLE
Can't open link with Enter using JAWS/IE11

### ISSUE THREAD

**Issue by dsr15** [FreedomScientific/VFO-standards-support#58]
### Summary

A link with some basic HTML that should be identified as a link does not activate when it has focus and enter is pressed using JAWS/IE11. The same link activates on enter fine with JAWS/Firefox. I'm using JAWS 18.

<a id="dmoTpl:cl1" class="xfi" onclick="return false;" href="#">Link Text</a>

### Expected result

Link can be activated on Enter key press.

### Actual result

Link isn't activated.

### Example

<a id="dmoTpl:cl1" class="xfi" onclick="return false;" href="#">Link Text</a>

#### JAWS version and build number

JAWS 18
 
#### Browser and version:

Internet Explorer 11

**Comment by stevefaulkner** [comment]
@dsr15 I tested with JAWS 2018 and IE 11 I cannot reproduce the issue you are describing.

**Comment by dsr15** [comment]
Interesting. What version of Windows were you using? I was using Windows 10 (touch).

**Comment by stevefaulkner** [comment]
@dsr15 Windows 10, desktop

**Comment by kboucher** [comment]
For anyone who comes across this issue, I was able to reproduce it when the link did NOT have an `href` attribute (but did have a JavaScript click handler attached). Since the example above includes an `href` attribute, it makes sense to me that this issue was closed as _can't reproduce_.

In our case, we simply added the `href` attribute to the `a` tag (and prevented event bubbling) and were then able to activate the link via the keyboard with JAWS running.

- Windows 10
- IE 11
- JAWS 2020

**Comment by stevefaulkner** [comment]
@kboucher note that a `<a>` element without a `href` is non interactive and does not represent a link

> If the `a` element has no `href` attribute, then the element represents a placeholder for where a link might otherwise have been placed, if it had been relevant, consisting of just the element's contents.
source: https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element
